src/main/java/com/team701/buddymatcher/controllers/users/UserController.java

```diff
@@ -56,4 +66,55 @@ public UserController(UserService userService, ModelMapper modelMapper) {
         }
     }
 
+    @Operation(summary = "Post method for a user logging in using Google")
+    @PostMapping(path = "/login", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> loginWithGoogle(HttpServletRequest request) throws GeneralSecurityException, IOException {
+        String CLIENT_ID = "158309441002-q8q49tjicngt1tp6p9t7ecvdrn9ar78j.apps.googleusercontent.com";
+        System.out.println("Made it");
```

Comment:
Could you please remove these System.out.println() statements before merging?

Reply:
Went through on a double-check and removed (hopefully) all the print statements :)

Reply:
I found one more, will comment on that one now :)